.github/workflows/conformance-bpf-tproxy.yaml

```diff
@@ -0,0 +1,156 @@
+name: Conformance BPF tproxy (ci-bpf-tproxy)
```

Comment:
I looks like this workflow is triggerd on PR and push and not via Ariane. So the trigger phrase is not needed, unless we want to change the workflow to be triggered by Ariane.

Reply:
Good catch - thanks! Indeed it was initially a ariane based workflow :)

changed